.github/workflows/publish-npm-packages.yml

```diff
@@ -48,11 +48,56 @@ jobs:
         run: |
           EXTENSIONS=$(node ${{ env.SCRIPTS_DIR }}/get-extensions.js)
           echo "::set-output name=extensions::{\"include\":${EXTENSIONS}}"
-  publish:
+  publish-core:
     if: github.repository == 'ampproject/amphtml'
     environment: bento_publishing
     needs: setup
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.events.inputs.amp-version }}
+      - uses: actions/setup-node@v2
+        with:
+          registry-url: https://registry.npmjs.org
+          node-version: 16
+      - name: Get latest scripts
+        run: |
+          wget -q "${{ env.SCRIPTS_REPO }}/${{ env.SCRIPTS_DIR }}/utils.js" -O ${{ env.SCRIPTS_DIR }}/utils.js
+          wget -q "${{ env.SCRIPTS_REPO }}/${{ env.SCRIPTS_DIR }}/get-core-version.js" -O ${{ env.SCRIPTS_DIR }}/get-core-version.js
+          wget -q "${{ env.SCRIPTS_REPO }}/${{ env.SCRIPTS_DIR }}/build-npm-binaries.js" -O ${{ env.SCRIPTS_DIR }}/build-npm-binaries.js
+      - name: Build package
+        run: |
+          npm ci
+          node ${{ env.SCRIPTS_DIR }}/build-npm-binaries.js
+          cd ./src/bento/core
+          npm version $(node ${{ env.SCRIPTS_DIR }}/get-core-version.js ${{ github.event.inputs.amp-version }}) --git-tag-version=false --allow-same-version=true
+      - name: Publish package for Nightly releases
+        if: github.event.inputs.tag != 'latest'
+        run: |
+          npm publish ./src/bento/core --access public --tag ${{ github.event.inputs.tag }}
```

Comment:
Do you need to `cd` back to root? Just checking

Reply:
i'm actually not sure that's what I was trying to ask [here](https://github.com/ampproject/amphtml/pull/37645#discussion_r814333215), but I can cd back to root

Reply:
resolved in linked discussion